MaxText/train.py

```diff
@@ -166,6 +167,34 @@ def save_checkpoint(checkpoint_manager, step, state, dataset_type='c4', data_ite
         items=orbax.checkpoint.args.PyTreeSave(item=state)
         ))
 
+def configure_upload_to_vertex_tensorboard(config, vertex_tensorboard_manager):
```

Comment:
does this feel like it should be in vertex_tensorboard.py?

Reply:
just a feeling

Reply:
Moved to VertexTensorboardManager class.